src/api/estimatedNextBatchTime.js

```diff
@@ -1,12 +1,12 @@
-const { get } = require('request-promise')
+const get = require('../lib/dvf/get-generic')
 
 module.exports = async dvf => {
   // avoid browser cache with timestamp as querystring
   const t = Date.now()
 
   const url = `${dvf.config.api}/v1/trading/r/estimatedNextBatchTime?t=${t}`
   try {
-    const data = await get(url)
+    const data = await get(dvf, url)
```

Comment:
generic-get prepends `dvf.config.api` to the 2nd arg so if you want to use it we should remove do:

```js
const url = `/v1/trading/r/estimatedNextBatchTime?t=${t}`
```

https://github.com/rhinofi/client-js/blob/a51ee785cf74e2de056cc7480d92b248c4b560a6/src/lib/dvf/get-generic.js#L6